### PR TITLE
Images not displaying correctly [YEP-128]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@bruskiza @miltontony ready for review

**Comment by bruskiza** [comment]
LGTM

File: tuneme/templates/core/article_page.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
   {% endif %}
 
   {% if self.image %}
-    {% image self.image width-98 as tmp_photo %}
+    {% image self.image width-620 as tmp_photo %}
```
Comment (miltontony) [inline tuneme/templates/core/article_page.html]: This is terrible for feature phone - you'll end up with a huge image downloaded for low end users

**Comment by smarzban** [comment]
@bruskiza ready for review

**Comment by bruskiza** [comment]
:+1:

**Pull request merged**